app/views/ops/_settings_rhn_edit_tab.html.haml

```diff
@@ -92,10 +92,10 @@
           = _("Organization")
         .col-md-8
           - if @edit[:organizations].length == 1
-            = @edit[:new][:customer_org]
+            = @edit[:new][:customer_org_display]
           - else
             = select_tag('customer_org',
-                          options_for_select((@edit[:new][:customer_org] ? [] : [["<Choose>", nil]]) + Array(@edit[:organizations].sort_by { |k, _v| k.downcase }),
+                          options_for_select((@edit[:new][:customer_org_display] ? [] : [["<Choose>", nil]]) + Array(@edit[:organizations].sort_by { |k, _v| k.downcase }),
```

Comment:
And down here we are saying "If customer_org_display is not nil, meaning we got it from the database, don't give the chooser option." And that also makes sense with the selected value of `@edit[:new][:customer_org]` which will be nil if we didn't get it from the database, effectively setting the `"<Choose>"` option as the default.

I can change this back also, I don't think it makes a difference.